### PR TITLE
Updating `arsenic.hackclub.com`

### PR DESCRIPTION
**Pull request by SahandSoleimani**
# Updating `arsenic.hackclub.com`

## Description

Adding MX & TXT records to enable Zoho Mail for `arsenic.hackclub.com`!

### CODE REVIEW

**Comment by SahandSoleimani** [comment]
Hi, I hope you're having a wonderful day!

After committing the updates, it said it has some conflicts that needed to be resolved, I resolved the final changes I wanted to exist and clicked on mark as resolved, now the "do a dry run" test on the octodns test failed, I'm wondering what the problem is and how I can fix it.

Thanks in advance!

File: hackclub.com.yaml [hunk 1]
```diff
@@ -436,8 +436,18 @@ archived.guide:
   value: hackclub-guide.netlify.com.
 arsenic:
   - ttl: 600
-    type: CNAME
+  - type: CNAME
```
Comment (Muirrum) [inline hackclub.com.yaml:439]: remove this hyphen to resolve the tests

```suggestion
    type: CNAME
```

**Review by Muirrum — CHANGES_REQUESTED** [review]

**Comment by SahandSoleimani** [comment]
Hi, I removed the hyphen but the test is still failing!

**Comment by SahandSoleimani** [comment]
"octodns.zone.InvalidNodeException: Invalid state, CNAME at arsenic.hackclub.com. cannot coexist with other records" this is the final error given in the "Do a dry run" test, any ideas what I can do?

**Comment by SahandSoleimani** [comment]
Alright so, I tried doing some things but it turns out I'm completely clueless.
It seems to me that it says I can't have any other records existing along with the cname record for `arsenic.hackclub.com`, so I just wanted to confirm this and close this pull request without merging if nothing else can be done.

Thanks in advance!

**Pull request closed without merging**